runtimes/pythonrt/build.go

```diff
@@ -0,0 +1,65 @@
+package pythonrt
+
+import (
+	"archive/tar"
+	"bytes"
+	"context"
+	"errors"
+	"io"
+	"io/fs"
+	"strings"
+
+	"go.autokitteh.dev/autokitteh/internal/kittehs"
+	"go.autokitteh.dev/autokitteh/sdk/sdktypes"
+	"go.uber.org/zap"
+)
+
+// TODO: Move build to runner
```

Comment:
this is @tebeka 's code?

Reply:
ENG-1563
refactor build system to use remote runners / builders